start_notebook.sh

```diff
@@ -141,8 +141,12 @@ else
     XENON_CONFIG_BIND=""
 fi
 
-SINGULARITY_COMMAND="singularity exec -e $XENON_CONFIG_BIND"
-SINGULARITY_COMMAND+=" $CONTAINER /bin/bash -c '$XENON_CONFIG_OVERRIDE $DIR/$INNER'"
```

Comment:
I am curious about why `CONTAINER` appeared twice: this line and https://github.com/XENONnT/env_starter/pull/64/files#diff-9aba0dbd9dd266e3f6cf095463c88efdb2c850251c10778c62cf2e4278209770L158

Reply:
It was a redundant one so is removed in this update